<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clamsSpeciesFix</class>
 <widget class="QDialog" name="clamsSpeciesFix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CLAMS Fix Species</string>
  </property>
  <widget class="QGroupBox" name="bottomBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>981</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="doneBtn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyChangeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply Fix!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear Filters</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Measurement values</string>
   </property>
  </widget>
  <widget class="QTableView" name="measureView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>621</width>
     <height>501</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>351</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="scientistBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>15</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="workstationBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>68</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="speciesBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>121</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>78</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Workstation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scientist</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>131</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Species</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Filter by</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>259</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="startIDBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start specimen_ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endIDBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>71</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>End specimen_ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="startIDLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>16</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The starting specimen ID the fix will be applied to.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="endIDLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>71</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The ending specimen ID the fix will be applied to.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="reassignTabs">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>130</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the &amp;quot;Species&amp;quot; tab a choose a species to assign to the specimen ids defined above. Select the &amp;quot;Sex&amp;quot; tab to choose a sex to assign to the specimen defined above.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="species">
     <attribute name="title">
      <string>Reassign Species</string>
     </attribute>
     <widget class="QComboBox" name="newSpeciesBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>38</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>8</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New Species Assignment</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sex">
     <attribute name="title">
      <string>Reassign Sex</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>8</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New Sex Assignment</string>
      </property>
     </widget>
     <widget class="QComboBox" name="newSexBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>38</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unsexed</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>223</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Reassign species or sex</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
